### DISCUSSION TITLE
Convert docker run to docker compose

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [louislam/dockge#529] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I see that Dockge can convert a run command to a compose file but I cannot work out where the convert option is found.

### 🐻 Dockge Version

1,4.2

### 💻 Operating System and Arch

Ubuntu 22.03

### 🌐 Browser

Edge

**Comment by JinbaIttai** [comment, marked as answer]
This one had me searching for a little while as well, because the text entry box kind of blends into the background

But it's on the main dockge page, right underneath the "active", "exited", "inactive" counters.

![dockge-run](https://github.com/louislam/dockge/assets/1313792/317dce8e-569a-45c2-91ed-9bc7b97f37b7)

**Comment by ZuluPilgrim** [comment]
Perfect; its obvious when  you see it :)
